test/e2e/config/ck8s-docker.yaml

```diff
@@ -51,9 +51,10 @@ providers:
           - old: "imagePullPolicy: Always"
             new: "imagePullPolicy: IfNotPresent"
     files:
-      - sourcePath: "../data/infrastructure-docker/cluster-template.yaml"
       - sourcePath: "../data/infrastructure-docker/cluster-template-kcp-remediation.yaml"
       - sourcePath: "../data/infrastructure-docker/cluster-template-md-remediation.yaml"
+      - sourcePath: "../data/infrastructure-docker/cluster-template-upgrades.yaml"
+      - sourcePath: "../data/infrastructure-docker/cluster-template.yaml"
```

Comment:
Just being curious, does the ordering matter here?

Reply:
I don't think so, these are just the specifications of template files.